testing/python/approx.py

```diff
@@ -420,14 +420,14 @@ def test_negative_tolerance(
     ) -> None:
         # Negative tolerances are not allowed.
         with pytest.raises(ValueError):
-            1.1 == approx(1, rel, abs)
+            1.1 == approx(1, rel, abs)  # noqa: B015
```

Comment:
Maybe ignore the rule for the entire file (IIRC you add `# ruff: noqa: B015` at the top of the file)

Reply:
bdd38f2a4f797c1066880c7a2ec6595ecd150e9c